integration_tests/Dockerfile-wsgi-patch

```diff
@@ -0,0 +1,17 @@
+FROM python:3.9-slim-bullseye
+
+RUN apt update && apt install --assume-yes gcc
```

Comment:
why do we need gcc?

Reply:
One of the required python library (`netifaces`) is using  C headers files during its installation

Reply:
Would be a good idea to comment that, that way we know why it's important and if it might not be anymore.